azure-quantum/azure/quantum/qiskit/backends/backend.py

```diff
@@ -46,6 +51,41 @@ def __init__(
         **fields
     ):
         super().__init__(configuration, provider, **fields)
+    
```

Comment:
Since this is the most "base" backend class, we should override it with our signature with the `shots` option.